milestone34.cabal

```diff
@@ -0,0 +1,45 @@
+cabal-version:   3.0
+name:            milestone34
+version:         0.0.0.1
+
+-- synopsis:
+-- description:
+license:         Apache-2.0
+license-file:    LICENSE
+author:          MLabs
+maintainer:      mlabs@mlabs.city
+
+-- copyright:
+build-type:      Simple
+extra-doc-files: CHANGELOG.md
+
+-- extra-source-files:
+
+common warnings
+  ghc-options: -Wall
+
+library
+  import:           warnings
+  exposed-modules:  MyLib
+
+  -- other-modules:
+  -- other-extensions:
+  build-depends:
+    , base         ^>=4.18.2.0
+    , plutus-tx
+
+  hs-source-dirs:   src
+  default-language: Haskell2010
```

Comment:
PlutusTx requires a bunch of GHC flags and extensions to properly work, and it'd be good if we could also add these.

Reply:
Added according to [this doc](https://plutus.readthedocs.io/en/latest/extensions-flags-pragmas.html#), plus `-fplugin-opt PlutusTx.Plugin:defer-errors`